src/clients/services-manager.rb

```diff
@@ -261,6 +261,21 @@ def toggle_service
     UI.SetFocus(Id(Id::SERVICES_TABLE))
     true
   end
+
+  def display_width
+    UI.GetDisplayInfo["Width"] || 80
+  end
+
+  def shortened_service_name(name)
+    return name if name.size < max_service_name
+
+    name[0..(max_service_name-3)] + "..."
```

Comment:
yeah, would be nice to have it somewhere :) I have just strong dislike to add it to Yast::String :)